microcosm_postgres/encryption/constants.py

```diff
@@ -0,0 +1,2 @@
+ENCRYPTION_V1_DEFAULT_KEY = "default"
```

Comment:
in the v1 encryption, they assumed that there would be a default with the key `default` that you configured using config.

in v2 encryption, I've assumed that we dynamically create the "default" encryptor to avoid having having to update each service's config. I've differentiated them both by using capital `DEFAULT`